dubbo-demo/dubbo-demo-protobuf/dubbo-demo-protobuf-consumer/pom.xml

```diff
@@ -0,0 +1,78 @@
+<!--
```

Comment:
we should move this demo into [dubbo-samples](https://github.com/apache/incubator-dubbo-samples)

Reply:
remove these demos.And these will be added in a pull request to dubbo-samples.